packages/lab/src/tabs-next/TabNext.tsx

```diff
@@ -0,0 +1,87 @@
+// TODO close button needs to be a button. Hence tab needs to include 2 buttons
+import { makePrefixer, useForkRef } from "@salt-ds/core";
+import {
+  forwardRef,
+  ForwardedRef,
+  MouseEvent,
+  ReactElement,
+  useCallback,
+  useRef,
+} from "react";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import { TabProps } from "./TabsNextTypes";
+
+import tabCss from "./TabNext.css";
+
+const withBaseName = makePrefixer("saltTabNext");
+
+export const TabNext = forwardRef(function Tab(
+  {
+    ariaControls,
+    children,
+    className,
+    index,
+    label,
+    onClick,
+    onFocus: onFocusProp,
+    onKeyDown,
+    onKeyUp,
+    onMouseDown,
+    selected,
+    tabIndex,
+    ...props
+  }: TabProps,
+  ref: ForwardedRef<HTMLDivElement>
+): ReactElement<TabProps> {
+  const targetWindow = useWindow();
+  useComponentCssInjection({
+    testId: "salt-tab-next",
+    css: tabCss,
+    window: targetWindow,
+  });
+  if (index === undefined || onClick === undefined || onKeyDown === undefined) {
+    throw Error(
+      "index, onClick, onKeyDown are required props, they would normally be injected by Tabstrip, are you creating a Tab outside of a Tabstrip"
+    );
+  }
+  const root = useRef<HTMLDivElement>(null);
+  const setForkRef = useForkRef(ref, root);
+  const handleClick = useCallback(
+    (e: MouseEvent) => {
+      e.preventDefault();
+      onClick(e, index);
+    },
+    [index, onClick]
+  );
+
+  return (
+    <div
+      {...props}
+      className={withBaseName()}
```

Comment:
I think we also need to pass `className`?

Reply:
✅